packages/cftime/patches/upper_case_cython.patch

```diff
@@ -0,0 +1,8 @@
+diff --color -Naur cftime-1.6.0.orig/pyproject.toml cftime-1.6.0/pyproject.toml
+--- cftime-1.6.0.orig/pyproject.toml	2022-03-03 10:59:04.000000000 -0300
++++ cftime-1.6.0/pyproject.toml	2022-05-05 10:11:01.523853642 -0300
+@@ -1,3 +1,3 @@
+ [build-system]
+-requires = ["setuptools>=41.2", "cython", "wheel", "oldest-supported-numpy"]
++requires = ["setuptools>=41.2", "Cython", "wheel", "oldest-supported-numpy"]
```

Comment:
Hmm, package names should probably be normalized in Pyodide.

Reply:
Looks like @hoodmane already have a solution for it. See https://github.com/pyodide/pyodide/issues/2503

Reply:
Well I didn't solve the capitalization problem. Some part of the pypabuild unisolation introduces case sensitivity I guess, but I don't understand how. The fix was not to unisolate cython which is an improvement because what we actually want is to always include cython. But we can always include cython by always including it and this is better than unisolating it.